pi-display/chase-kiosk-helper.py

```
@@ -0,0 +1,1191 @@
+    the upstream connection. Driver/chipset-dependent; `iw` has no
+    machine-readable output for this, so we scrape `iw phy <N> info`'s
+    "valid interface combinations" section. Any parse failure returns
+    False — safer to fall back to the single-radio flow than to attempt
+    something that may not actually be supported.
+    """
+    phy = _wlan_phy()
+    if not phy:
+        return False
+    try:
+        res = _iw("phy", phy, "info")
+    except Exception:
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)